index.bs

```diff
@@ -2758,44 +2793,69 @@ Appeal by Advisory Committee Representatives</h3>
 
 	An [=Advisory Committee representative=] initiates an [=appeal=] by sending a request to the [=Team=],
 	and should also share this request with the [=Advisory Committee=].
-	The request should say “I appeal this Decision”
+	The request <em class=rfc2119>should</em> say “I appeal this Decision”
 	and identify the decision,
-	and may also include their rationale for appealing the decision.
+	and <em class=rfc2119>may</em> also include their rationale for appealing the decision.
 
 	Note: See [[DECISION-APPEAL inline]] for a recommendation
 	on how to communicate an appeal request to the [=Team=] and the [=AC=].
 
-	Within one week the [=Team=] <em class="rfc2119">must</em> announce the appeal process
+	Within one week the [=Team=] <em class="rfc2119">must</em> initiate the [=appeal=]
+	in the form of a [=Advisory Committee Override=]
+	on the proposal to overturn the decision.
+
+	The conclusion of the [=Advisory Committee Override=] is final:
+	the same decision cannot be appealed more than once.
+
+	When the [=Advisory Committee Override=]
+	approves the proposal to overturn the decision,
+	the proposal <em class=rfc2119>may</em> be returned for additional work,
+	after which the applicable decision process is re-initiated.
+
+<h3 id="ac-override">
+Advisory Committee Override</h3>
+
+	An <dfn export>Advisory Committee Override</dfn> is an exceptional two-step procedure
+	used to resolve certain matters where the usual [=consensus=] process
+	and its [[#addressing-fo|escalation path]] are insufficient.
```

Comment:
```suggestion
	and its [[#addressing-fo|escalation path]] have been proven to be insufficient.
```

Reply:
I think that phrasing doesn't work well for the AB/TAG recall case. Make me wonder about my phrasing too, to be honest.

Reply:
Resolved by https://github.com/w3c/process/pull/888#discussion_r1657569297